stylus/components/badge.styl

```diff
@@ -0,0 +1,38 @@
+@require '../settings/palette'
+
+radius = .4375rem
```

Comment:
I was confused about the name of this `radius` variable and saw that it's not used for radius at all.
I wonder if it wouldn't be best to name it something like `size` and prefer something like this:
```
size = .875rem
[…]
bottom - size / 2
right - size / 2
width size
height size
```

Reply:
Yeah the idea behind `radius` was more the radius of the circle. I understand it can be confusing, I change it.